WebDevStudios/wd_s
# Fixes an issue where not all partials are imported

### DESCRIPTION

Noticed this while troubleshooting #758. For some reason, not all of the .scss partials were being imported. This solves that!

### OTHER


### STEPS TO VERIFY

Just run an `npm run start` or `npm run build` and verify that changes in all .scss files are being compiled.

### DOCUMENTATION

Will this pull request require updating the wd_s [wiki](https://github.com/WebDevStudios/wd_s/wiki)? – No changes
